app/views/search/_form.html.erb

```diff
@@ -225,4 +225,17 @@ end
   </div>
 </form>
 
+<% if Flipflop.enabled?(:boolean_picker) %>
+  <% if cookies[:boolean_type].present? %>
+    Your boolean preference for this session is: <%= cookies[:boolean_type] %>
```

Comment:
I ended up going with our existing info panel layout but used the `<aside>` as you suggested rather than `div` as the top level element. I don't love it, but it is definitely better than it was and feels good enough for a fake-ish feature :) 

Thanks for suggesting we add some markup/styles as it will be better to work from this than the nonsense I had in there if we resurrect this!

Reply:
Ha! I forgot how loud our panel styling is in this app. Much more readable now, though -- thanks!